command/command_linux.go

```diff
@@ -9,12 +9,14 @@ import (
 )
 
 func specGenerators() []spec.Generator {
+	instanceGenerator, _ := specLinux.NewInstanceGenerator("")
```

Comment:
I don't think it's good to discard errors. Let's not include the generator to the result if any error is returned.